### ISSUE TITLE
Request Data group information for PRJNA943687

### ISSUE THREAD

**Issue by mengqingren** [qsu123/ASD_multi-kingdom_diagnosis#1]
Dear Doctor,
Could you provide the sample grouping information, such as whether S0001 corresponds to ASD or ND? I am looking forward to your reply.

**Comment by ivagljiva** [comment]
I would also be interested in the sample group information and would like to second this request. 

The publicly available sequences at [PRJNA943687](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA943687) are an excellent resource for the community and could be useful for investigating several hypotheses if the correspondence between samples and their groups are known. We would appreciate the sharing of this essential metadata. :)

**Comment by cssmillie** [comment]
We are also interested in obtaining the sample metadata for PRJNA943687.

@ivagljiva @mengqingren @meren were you able to obtain it?

**Comment by meren** [comment]
No, @cssmillie. We haven't been able to obtain it. The lack of response on GitHub makes me think that perhaps no one is checking the notifications from GitHub. We will try email, and report back :)

**Comment by qsu123** [comment]
@meren @cssmillie @ivagljiva @mengqingren Hi all, I am so sorry for I didn't notice the comments here. We are now updating the PRJNA943687 and the metadata will be available soon. I will let you know when it is ready. Sorry again for any inconvenience caused by this.

**Comment by ivagljiva** [comment]
Hello @qsu123 , just checking in to see if the data and metadata are available yet? :)

**Comment by qsu123** [comment]
Yes. All the metagenome were released in the PRJNA943687 last month. Please contact the Corresponding author, Prof. Siew Ng (siewchienng@cuhk.edu.hk) for group information.
